test/verify/check-superuser

```diff
@@ -182,6 +182,27 @@ session include system-auth
         b.wait_not_present(".pf-c-modal-box:contains('Administrative access')")
         b.wait_text("#super-user-indicator", "Limited access")
 
+    def testRemoveBridgeConfig(self):
+        b = self.browser
+        m = self.machine
+
+        m.needs_writable_usr()
```

Comment:
Totally not important, but FYI: Since today's commit 413111f230ccabaaded2b63f081472ffedd3fccd , tests can use `self.restore_dir()` to get a writable directory also on OSTree.